readthedocs/projects/views/public.py

```diff
@@ -119,7 +120,8 @@ def _badge_return(redirect, url):
 
 
 # TODO remove this, it's a temporary fix to heavy database usage
-@cache_page(60 * 30)
+@cache_page(60 * 5)
```

Comment:
We can probably just get rid of this caching now, I've fixed the index misses that were bogging down the database.